bin/plot_krun_results

```diff
@@ -344,17 +356,26 @@ class ProcessExecChart(object):
         else:
             self.common = None
         # Marshal changepoint data.
-        self.changepoints = changepoints
-        self.changepoint_means = changepoint_means
+        self.changepoints = list()
+        self.changepoint_means = [changepoint_means[0]]
+        self.changepoint_vars = [changepoint_vars[0]]
+        self.delta = classifier['delta']
         self.classification = classification
-        if self.changepoint_means:
+        if changepoint_means:
+            if changepoints:
+                self.last_changepoint = changepoints[-1]
+            else:
+                self.last_changepoint = None
+            self.last_segment_mean = changepoint_means[-1]
+            self.last_segment_var = changepoint_vars[-1]
             self.segment_means = list()  # ((x0, y0), (x1, y1)) pairs.
-            if self.x_bounds[0] > 0:
-                for index, changepoint in enumerate(self.changepoints):
-                    if changepoint < self.x_bounds[0]:
-                        del self.changepoint_means[index]
-                        del self.changepoints[index]
+            for index, changepoint in enumerate(changepoints):
```

Comment:
This is potentially unsafe, as we're also adding items to `changepoints` in the loop. Are those new things supposed to be considered by this loop as well? If not, it would be clearer to the reader if a new list was made inside the `for` loop, and then `changepoints.extend(newlist)` after the loop.